### PR TITLE
use angle bracket syntax

### PR DESCRIPTION
**Pull request by the-bionic**
Update to use Angle Bracket syntax

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -162,7 +162,7 @@ You can pass any custom pikaday option through the component like this
 
 ```handlebars
 <label>
-  {{pikaday-input options=(hash numberOfMonths=2 disableWeekends=true disableDayFn=(action 'someAction'))}}
+  <PikadayInput @options={{hash numberOfMonths=2 disableWeekends=true disableDayFn={{action 'someAction'}}}}/>
```
Comment (efx) [inline README.md]: I think we still need to use `(action 'someAction')` because it is a nested handlebars expression.

Comment (the-bionic) [reply README.md]: Thanks for catching this 👍 update made! :)

File: README.md [hunk 2]
```diff
@@ -33,8 +33,8 @@ You can also pass in other closure actions to handle `onOpen`, `onClose` and `on
 ```handlebars
 <label>
   Start date:
-  {{pikaday-input onOpen=(action 'doSomethingOnOpen') onClose=(action 'doSomethingOnClose')
-    onDraw=(action 'doSomethingOnDraw')}}
+  <PikadayInput @onOpen={{action 'doSomethingOnOpen'}} @onClose={{action 'doSomethingOnClose'}}
+    onDraw={{action 'doSomethingOnDraw'}}/>
```
Comment (efx) [inline README.md]:
```suggestion
    @onDraw={{action 'doSomethingOnDraw'}}/>
```

Comment (efx) [reply README.md]: @the-bionic last tweak and I'll merge!

Comment (the-bionic) [reply README.md]: Woah, thanks for catching this! 👍

**Comment by efx** [comment]
Grand, thanks again @the-bionic!

**Pull request merged**